pkg/hub/controllers/placemanifestworkcontroller/placemanifestwork_deploy_reconcile.go

```diff
@@ -18,5 +30,133 @@ type deployReconciler struct {
 
 func (d *deployReconciler) reconcile(ctx context.Context, pw *workapiv1alpha1.PlaceManifestWork) (*workapiv1alpha1.PlaceManifestWork, reconcileState, error) {
 	// TODO the main manifestwork create/update/delete logic, we may also need to manage the status condition for whether manifestwork is created here
-	return pw, reconcileContinue, nil
+	klog.Info("reconciling placeManifestWork ", pw.Name)
+
+	placement, err := d.placementLister.Placements(pw.Namespace).Get(pw.Spec.PlacementRef.Name)
+	if err != nil {
+		if errors.IsNotFound(err) {
+			apimeta.SetStatusCondition(&pw.Status.Conditions, GetPlacementDecisionVerified(workapiv1alpha1.PlacementDecisionNotFound, ""))
+		}
+		return pw, reconcileContinue, fmt.Errorf("Failed get placement %w", err)
+	}
+
+	//klog.Info("my reconciling placement ", placement)
+	errs := []error{}
+	req, err := labels.NewRequirement(PlaceManifestWorkControllerNameLabelKey, selection.In, []string{pw.Name})
+	if err != nil {
+		errs = append(errs, err)
+	}
+
+	selector := labels.NewSelector().Add(*req)
+	manifestWorks, err := d.manifestWorkLister.List(selector)
+	if err != nil {
+		errs = append(errs, err)
```

Comment:
same, directly return err, you would not want the following code to proceed if list here returns err.